code/game/machinery/teleporter.dm

```diff
@@ -192,25 +200,53 @@
 
 
 /obj/machinery/teleport/hub
-	name = "teleporter hub"
-	desc = "The teleporter hub handles all of the impossibly complex busywork required in instant matter transmission."
-	icon_state = "tele0"
-	dir = EAST
+	name = "teleporter pad"
+	desc = "The teleporter pad handles all of the impossibly complex busywork required in instant matter transmission."
+	icon_state = "pad"
 	idle_power_usage = 10
 	active_power_usage = 2000
 	var/obj/machinery/computer/teleporter/com
+	light_color = "#02d1c7"
+
+/obj/machinery/teleport/hub/Initialize()
+	..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/machinery/teleport/hub/LateInitialize()
```

Comment:
Might be worth just commenting here that you LateInitialize() so the hub and station have a chance to connect, otherwise it looks a bit pointless.